Indicators/PythonIndicator.cs

```diff
@@ -44,71 +44,80 @@ private static string GetIndicatorName(PyObject indicator)
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the PythonIndicator class using the specified name.
+        /// </summary>
+        /// <remarks>This overload allows inheritance for python classes with no arguments</remarks>
+        public PythonIndicator()
+            : base("")
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the PythonIndicator class using the specified name.
+        /// </summary>
+        /// <remarks>This overload allows inheritance for python classes with multiple arguments</remarks>
+        public PythonIndicator(params PyObject[] args)
+            : base (GetIndicatorName(args[0]))
+        {
+        }
+
         /// <summary>
         /// Initializes a new instance of the PythonIndicator class using the specified name.
         /// </summary>
         /// <param name="indicator">The python implementation of <see cref="IndicatorBase{IBaseDataBar}"/></param>
         public PythonIndicator(PyObject indicator)
             : base(GetIndicatorName(indicator))
+        {
+            SetIndicator(indicator);
+        }
+
+        /// <summary>
+        /// Sets the python implementation of the indicator
+        /// </summary>
+        /// <param name="indicator">The python implementation of <see cref="IndicatorBase{IBaseDataBar}"/></param>
+        public void SetIndicator(PyObject indicator)
         {
             using (Py.GIL())
             {
-                foreach (var attributeName in new[] {"Update", "IsReady", "Value"})
+                foreach (var attributeName in new[] {"IsReady", "Update", "Value"})
```

Comment:
Curious, why change the ordering here?

Reply:
I changed it because `IsReady` is more likely to be not implemented.